src/positiondetector/SimpleThreshold.h

```diff
@@ -54,18 +54,20 @@ class SimpleThreshold : public PositionDetector {
     void set_max_object_area(double value) { max_object_area_ = value; }
     void set_erode_size(int erode_px);
     void set_dilate_size(int dilate_px);
+    void set_mincomp_size(int mincomp_val);
 
 private:
 
     // Intermediate variables
     cv::Mat threshold_frame_;
+    cv::Mat nonmasked_frame_;
 
     // Object detection
     double object_area_ {0.0};
 
     // Sizes of the erode and dilate blocks
-    int erode_px_ {0}, dilate_px_ {0};
-    bool erode_on_ {false}, dilate_on_ {false};
+    int erode_px_ {0}, dilate_px_ {0}, mincomp_val_ {0};
```

Comment:
int mincomp_ {0} on its own line with a an explanatory comment. It is not related to erode an dilate.